plugins/modules/redshift.py

```diff
@@ -170,11 +173,22 @@
       - Whether the cluster should have enhanced VPC routing enabled.
     default: false
     type: bool
+  tags:
+    description:
+      - A dictionary of resource tags.
+    type: dict
+    aliases: ['resource_tags']
+    version_added: "2.10"
```

Comment:
```suggestion
    version_added: "1.2.0"
```

And for brand new features this needs to use the next collection version.